BayerischeStaatsbibliothekOpacPlus.js

```diff
@@ -0,0 +1,37 @@
+{
+	"translatorID": "6b09b92d-f47e-4573-8a00-74245f7c6c5e",
+	"label": "Bayerische Staatsbibliothek (OpacPlus)",
+	"creator": "bkroll",
+	"target": "^https://opacplus.bsb-muenchen.de/metaopac/singleHit.do.*",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2017-07-02 19:51:29"
+}
+
+function detectWeb(doc, url) {
+	mediaTypeIcon = doc.getElementById("mediaTypeIcon");
+	mediaTypeNode = mediaTypeIcon.getElementsByTagName("img");
+	mediaType = mediaTypeNode[0].getAttribute("alt");
+
+	if (mediaType == "Buch") {
+		return "book";
+	}
+}
+
+/** get MarcXml representation from opac dataset and use MarcXml translator **/
+function doWeb(doc, url) {
+	marcxmlLinkNode = doc.getElementById("marcLink");
+	marcxmlLinkString = marcxmlLinkNode.getAttribute("href");
+	Zotero.Utilities.doGet("https://opacplus.bsb-muenchen.de" + marcxmlLinkString, translateMarcxml);
```

Comment:
`doGet` will handle a relative URL fine, so no need to concatenate it with the domain here.